hash/32x.xml

```diff
@@ -63,6 +63,7 @@
 	</software>
 
 	<software name="swa" supported="no">
+	<!-- Black screen on startup with both the DRC and the interpreter. -->
```

Comment:
When adding comments to XML, could you please indent them to block level?  This convention is used in all our software lists.

For example for this one it should look like:

```
    <software name="swa" supported="no">
        <!-- Black screen on startup with both the DRC and the interpreter. -->
        <description>Star Wars Arcade (Euro)</description>
```